src/lib/import_export/csv_parser.cpp

```diff
@@ -34,14 +34,25 @@ std::shared_ptr<Table> CsvParser::parse(const std::string& filename, const std::
     _meta = *csv_meta;
   }
 
+  _escaped_linebreak = std::string(1, _meta.config.delimiter_escape) + std::string(1, _meta.config.delimiter);
```

Comment:
Reading `delimiter` in the context of a line break made me wonder. The config has a `delimiter` which marks the end of a line and a `separator` that marks the end of a field. I believe in every other program, the entities indicating the end of a field are called `delimiters`. Should we rename that?

Reply:
Probably. That and a dozen other things in the CsvParser. Doesn't really make sense to tackle one (c/p) isolated thing in this PR.

Reply:
I guess you already maintain a private list. Do you want to publish it?

Reply:
No, I don't really do. We can have a quick chat about rewriting the damn thing at some point, if you want to.

Reply:
Sure.